### PR TITLE
#26 added contact messages page code formatted

### PR DESCRIPTION
**Pull request by chandra-pro**
Please review it and let me know.

### CODE REVIEW

File: components/messages/ChatHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        style={styles.backButton}
+        onPress={(): void => navigation.navigate('Contacts')}
+      >
+        <AntDesign name="arrowleft" size={24} color={theme.colors.white} />
+      </TouchableOpacity>
+      <View style={styles.profileOptions}>
+        <View style={styles.item}>
+          <View style={styles.avatar}>
+            <Text
+              style={{
```
Comment (mdshamoon) [inline components/messages/ChatHeader.tsx:25]: Please use a class here.

File: components/messages/ChatHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                fontSize: 25,
+                fontWeight: 'bold',
+                color: '#073F24',
+              }}
+            >
+              {name.charAt(0)}
+            </Text>
+          </View>
+          <View
+            style={{
```
Comment (mdshamoon) [inline components/messages/ChatHeader.tsx:39]: here as well

File: components/messages/ChatInput.tsx [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+            <Icon
+              name="paperclip"
+              size={23}
+              color={theme.colors.description}
+              style={{ transform: [{ rotate: '50deg' }] }}
+            />
+          </TouchableOpacity>
+        </View>
+        <TouchableOpacity style={styles.sendButton}>
+          <View style={{ justifyContent: 'center', alignItems: 'center', flexDirection: 'row' }}>
```
Comment (mdshamoon) [inline components/messages/ChatInput.tsx:84]: Please check all the places where inline style is used and replace them with classes

File: components/messages/MessageList.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
+import React, { useState, useRef } from 'react';
+import { ScrollView } from 'react-native';
+
+import Message from './Message';
+
+import { theme } from '../../theme';
+
+const MessagesList = ({ onSwipeToReply }: any) => {
+  const [messages, setMessages] = useState([
+    {
```
Comment (mdshamoon) [inline components/messages/MessageList.tsx:10]: Create a const fakeMessages and import here

File: theme.js
Comment (mdshamoon) [inline theme.js:1]: There is already a colors.js file defined in constants. Can you please reuse that.

**Review by mdshamoon — CHANGES_REQUESTED** [review]
Nice work. Overall looks good, just left some minor comments. Please remove the emoji files and code. We don't want people to make edits here and will use a library so that we follow standards for these.

**Comment by mdshamoon** [comment]
@chandra-pro also please don't create separate Pull request for the same issue. You can update it in the same branch,

File: components/messages/MessageList.tsx (hunk 4 above)
Comment (chandra-pro) [reply components/messages/MessageList.tsx:10]: I have already made fakemessages in MessageList.

File: components/messages/ChatHeader.tsx (hunk 1 above)
Comment (chandra-pro) [reply components/messages/ChatHeader.tsx:25]: resolved.

**Review by mdshamoon — APPROVED** [review]
LGTM 👍. Will remove the emoji part for now as it is getting stuck on my phone while testing

**Pull request merged**